src/lib/components/EventQAndA/index.js

```diff
@@ -0,0 +1,80 @@
+import {BaseElement} from '../BaseElement';
+import './_styles.scss';
+
+/**
+ * Filters out Q&A's based on `data-category` property.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class EventQAndA extends BaseElement {
+  constructor() {
+    super();
+    this.selectCategory = this.selectCategory.bind(this);
+
+    this.categories = new Set();
+    this.categorySelectElement = document.createElement('select');
+    this.categorySelectElement.classList.add('w-select--borderless');
+    this.categorySelectElement.addEventListener('change', this.selectCategory);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.addDropDown();
+
+    this.querySelectorAll('[data-category]').forEach((element) => {
+      element.addEventListener('click', this.closeDetail);
+    });
+  }
+
+  disconnectedCallback() {
+    this.querySelectorAll('[data-category]').forEach((element) => {
+      element.removeEventListener('click', this.closeDetail);
+    });
+    super.disconnectedCallback();
+  }
+
+  addDropDown() {
```

Comment:
this will run every time the element is connected. Which, to be fair, is only once right now. Maybe we can have a boolean or see if `this.categories.size` is truthy?